scripts/kubernetes/sth-pod.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sth
+  labels:
+    app: sth
+    app.kubernetes.io/name: sth
+spec:
+  serviceAccountName: sth-sa
+  automountServiceAccountToken: true
+  containers:
+  - name: sth
+    image: scramjetorg/sth:0.18
+    command: ["scramjet-transform-hub"]
+    env:
+      - name: DEVELOPMENT
+        value: \"false\"
+    args:
+      - "--runtime-adapter=kubernetes"
+      - "--k8s-runner-image=scramjetorg/runner:0.18"
+      - "--k8s-namespace=default"
```

Comment:
Is this the same namespace where sth will be running? if so, I suggest to add namespace to pod/deployment manifest or/and update README in case someone is using context with different namespace.